### DISCUSSION TITLE
What is the path this forks will take?

### DISCUSSION THREAD

**Discussion by Jaksor23** [Mr-X-GTA/YimMenu#18] [General]
With the recent clarification with YimMura and their decision to go back from closed source to YimMenu maybe forever, I would like to know what is the path that this fork will take, since it is the one that since the official one went off the air, continued to be updated and some of us are worried about our forks, since we depend in some way on you to keep our forks updated and besides that many of us want to do PR pending that we had with YimMenu.

**Comment by YimMenuTheBest** [comment]
@Mr-X-GTA Any bit of information for those of us who want to continue with our forks, but depend on pointer updates or reverse engineering from greats like you?
